SensusService/Protocol.cs

```diff
@@ -1012,6 +1121,50 @@ public void Start()
             }
         }
 
+        public void ScheduleStart()
+        {
+            DateTime startTime = new DateTime(_startTime.Year, _startTime.Month, _startTime.Day, _startTime.Hour, _startTime.Minute, 0);
+            TimeSpan timeUntilStart = startTime - DateTime.Now;
+            ScheduledCallback startProtocolCallback = new ScheduledCallback((callbackId, cancellationToken, letDeviceSleepCallback) =>
+            {
+                return Task.Run(() =>
+                {
+                    Start();
+                });
+
+            }, "Start protocol", null, "Protocol started.");
```

Comment:
The user notification message should be something like "Please open to start this study."